documentation/components/tutorial-flow-shortcut.asciidoc

```diff
@@ -103,10 +103,8 @@ components, `Scope`. If the user is typing something in one of the inputs and
 presses _Escape_, both input fields will be cleared and focus returned to the
 first field.
 
-This is useful in situations where the application has multiple
-instances of
-the same component visible at the same time on the screen and each component has
-the same shortcut.
+This is useful in situations where the same action needs to be triggered
+by a shortcut configured on all fields contained inside the same scope.
```

Comment:
"-- but not outside of the enveloping component." 

Would that be good addition?